client/app.jsx

```diff
@@ -23,102 +22,58 @@ class App extends React.Component {
     this._isMounted = false;
   }
 
-  fetchData() {
-    axios.get('/products')
-      .then((response) => {
-        console.log('this is our initial project data:', response.data)
 
-        this._isMounted && this.setState({
-          data: response.data,
-          currentItem: response.data[0],
-          currentItemId: response.data[0].id,
-          averageStars: null
-        })
-      })
-      .then(() => {
-        axios.get(`/reviews/${this.state.currentItemId}&count=1000`)
-          .then((response) => {
-            console.log('found our reviews data!', response.data.results)
-            this._isMounted && this.setState({
-              numberOfReviews: response.data.results.length,
-              reviewData: response.data.results
-            });
-          })
-          .catch((error) => {
-            console.log('error getting our response from styles get: ', error)
-          })
-        axios.get(`/reviews/meta/${this.state.currentItemId}`)
-          .then((response) => {
-            // console.log('response ratings', response.data.ratings);
-            var rateObj = response.data.ratings;
-            var result = 0;
-            var numRating = 0;
-            // console.log('result: ', result);
-            // console.log('rateObj: ', this.state.ratingObj);
-            for (var key in rateObj) {
-              // console.log('numKey');
-              result = result + Number(key) * Number(rateObj[key]);
-              numRating = numRating + Number(rateObj[key]);
-            }
-            // console.log('result: ', result);
-            // console.log('numRating: ', numRating);
-            var currRating = result / numRating;
-
-            this._isMounted && this.setState({
-              averageStars: currRating
-            })
-            //  console.log('state check of averageStars: ', this.state.averageStars)
-          })
-
-          .catch((error) => {
-            console.log('error inside reviews meta get: ', error)
-          })
-      })
-
-      .catch((error) => {
-        console.log('error in app.jsx axios get request, error:', error)
-      })
-  }
+componentDidMount() {
 
-  componentDidUpdate() {
-    //console.log('we are checking for updates YOOOOOOOOOOOOOOOOOOOOOOOOOOOOO')
+  if(!localStorage['websiteTraffic']) {
 
+    localStorage['websiteTraffic'] = {
+      'Overview': '',
+      'Related Items And Comparisons': '',
+      'Questions and Answers': '',
+      'Reviews and Ratings': ''
+    }
```

Comment:
This is a cool use of localStorage! Can't wait to see further functionality of that